README.md

```diff
@@ -1,18 +1,65 @@
-# is-this-an-mlm
```

Comment:
please:
- keep the old readme
- in the old readme already add a line like "made with sidebase" and then link sidebase